game/terrain.cpp

```diff
@@ -671,11 +671,7 @@ void Terrain::loadBaseSprite(const QString & spriteID, qint32 frameTime, qint32
         addChild(pSprite);
         m_terrainSpriteName = spriteID;
         QImage img;
-        if (QFile::exists(m_terrainSpriteName))
```

Comment:
Please revert this.
It has no disadvantage for linux but adds more options to windows

Reply:
This should not change the behavior at all on Windows, because by default "getUserPath()" returns an empty string (effectively acting the same as if the call was just `QFile::exists(m_terrainSpriteName)`. On Linux, it avoids it trying to read files from an unpredictable current directory.

Is there something I'm missing?

Reply:
Yes in certain cases the shaders aren't needed mainly when running without ui where the ui window still exists but is never shown. -> For the user of the game not relevant for me as a developer it's.

Reply:
I'll revert it for now, then. Looking at how many places there's code like this, I think all this stuff should eventually be unified into one utility function, and we can deal with these kinds of complexities better there.